.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist
+test
```

Comment:
do we need to ignore this?

Reply:
the dist folder yes, im not sure about the tests one, got told it was ok to at one point